src/vs/workbench/contrib/notebook/browser/view/renderers/backLayerWebView.ts

```diff
@@ -1402,16 +1402,20 @@ export class BackLayerWebView<T extends ICommonCellInfo> extends Themable {
 			createOnIdle: createOnIdle
 		} as const;
 
+		const transfer: ArrayBuffer[] = [];
+
 		let message: ICreationRequestMessage;
 		let renderer: INotebookRendererInfo | undefined;
 		if (content.type === RenderOutputType.Extension) {
 			const output = content.source.model;
 			renderer = content.renderer;
 			const first = output.outputs.find(op => op.mime === content.mimeType)!;
 
-			// TODO@jrieken - the message can contain "bytes" and those are transferable
-			// which improves IPC performance and therefore should be used. However, it does
-			// means that the bytes cannot be used here anymore
+
+			// Copy the underlying buffer so we only send over the data we need
```

Comment:
@jrieken @alexdima In this instance, `first.data.buffer` is a `Buffer` which looks like a `Uint8Array`. This `Buffer` is much larger than the `VSBuffer` for `first.data`

Is there a way to avoid this size mismatch? I suspect it's an optimization to reduce the number of memory allocations